burnman/seismic.py

```diff
@@ -45,7 +45,7 @@ def evaluate_all_at(self, depth_list):
     
 
 
-    def internal_depth_list(self):
+    def internal_depth_list(self,mindepth=0.,maxdepth=6371.e3):
```

Comment:
With this you can get the depth list for e.g. part of the PREM model instead of the entire earth.